sklearn/utils/tests/test_validation.py

```diff
@@ -108,6 +109,7 @@ def test_ordering():
     for copy in (True, False):
         Y = check_array(X, accept_sparse='csr', copy=copy, order='C')
         assert_true(Y.data.flags['C_CONTIGUOUS'])
+    """
```

Comment:
Please delete that code block rather than commenting it out with the multiline string markers.